remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
+                },
+                {
+                    "message": "You can use a more lively `ddev list` and `ddev describe` output with `ddev config global --table-style=bright`. Try it out!"
+                },
+                {
+                    "message": "DDEV's fiscal entity is now the DDEV Foundation. http://ddev.com/foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy."
```

I’d let the message read in its entirety before including the link as a way to go and learn more. (And use an HTTPS URL.)

```suggestion
                    "message": "DDEV's fiscal entity is now the DDEV Foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy. https://ddev.com/foundation"
```